packages/kit/src/runtime/app/forms.js

```diff
@@ -110,11 +113,25 @@ export function enhance(form_element, submit = () => {}) {
 	 */
 	const fallback_callback = async ({ action, result, reset }) => {
 		if (result.type === 'success') {
-			if (reset !== false) {
-				// We call reset from the prototype to avoid DOM clobbering
-				HTMLFormElement.prototype.reset.call(form_element);
-			}
-			await invalidateAll();
+			await client._invalidate_all_with_callback(() => {
```

Comment:
do we need to use `client_method` here? @gtm-nayan

Reply:
No, client should have been initialized by this point.